bessctl/conf/samples/mpls_push.bess

```diff
@@ -0,0 +1,11 @@
+import scapy.all as scapy
+
+eth = scapy.Ether(src='02:1e:67:9f:4d:ae', dst='06:16:3e:1b:72:32')
+ip = scapy.IP(src='192.168.0.1', dst='10.0.0.1')
+udp = scapy.UDP(sport=10001, dport=10002)
+test_packet = bytes(eth/ip/udp)
+
+mpls_push::MPLSPush()
```

Comment:
Besides this runtime command, I think the module should allow initial configuration. Please refer to `vlan_push.cc` as an example.